engine/movie/oak_speech/oak_speech.asm

```diff
@@ -159,6 +165,7 @@ OakSpeechText1:
 	text_end
 OakSpeechText2:
 	text_far _OakSpeechText2A
+	; The cry played does not match the sprite displayed.
```

Comment:
We've been pretty lenient about documenting "bugs" to include inconsistencies and oversights.
```suggestion
	; BUG: The cry played does not match the sprite displayed.
```